wrappers/wine-msvc.sh

```diff
@@ -80,4 +81,28 @@ else
 	sed -e "$dos2unix" -e "$unixify_path" -e "$unixify_line" <$WINE_MSVC_STDOUT &
 	sed -e "$dos2unix" -e "$unixify_path" <$WINE_MSVC_STDERR >&2 &
 	WINEDEBUG=-all wine64 "$EXE" "${ARGS[@]}" &>/dev/null
+	ec=$?
 fi
+
+[ $ec -ne 0 ] && exit $ec
+
+# Postprocess
+for a in "${ARGS[@]}"; do
+	case $a in
+		[-/]P) arg_P=$a ;;
+		[-/]Fi[zZ]:*) a=${a:5}; arg_Fi=${a//\\//} ;;
```

Comment:
This had me confused for a while, as I didn't expect callers to be calling us with arguments like `/Fiz:/home...`, but then I realize this runs on the translated arguments in `$ARGS`, so `z:/...` is indeed to be expected for absolute paths.

Would it make more sense to do this interpretation on the original unaltered arguments from before? I guess it doesn't matter much since it's easy to match these cases anyway.

Also, for the expansion `arg_Fi=${a//\\//}`, I don't quite see exactly which bash parameter substitution mechanism that uses - I presume it is related to translating backslashes to forward slashes?

Do we expect that to happen? I would expect callers to only call us with unix style paths here, and the translation above only injects `z:` but shouldn't convert to backslashes, right?

Reply:
> I would expect callers to only call us with unix style paths here

Yes, this makes the expansion unnecessary.

Actually `/...` is also supported by wine/windows (relative to current drive). Did you encounter any issue without the translation to `z:/...`?

Reply:
I believe it was mostly needed to disambiguate paths from options, e.g. `/home/foo` can be interpreted as the option `/h` with argument `ome/foo`, while `z:/home/foo` makes it clear that it is a path, not an argument.

Reply:
OK, fair enough.